s3/pkg/datastore/yig/config/config.go

```diff
@@ -0,0 +1,155 @@
+package config
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+type Config struct {
+	Endpoint   EndpointConfig
+	Log        LogConfig
+	StorageCfg StorageConfig
+	Database   DatabaseConfig
+}
+
+func (config *Config) Parse() error {
+	endpoint := viper.GetStringMap("endpoint")
```

Comment:
can we make a ConfigHelper or ConfigReader (that internally uses viper), so we can use the power of viper across OSDS, for all our configs ?

Reply:
I can make the changes in existing code if we are okay with the concept of using viper across OSDS

Reply:
and using viper is fine:)